docs/organizations/managing-people.md

```diff
@@ -1,6 +1,8 @@
 # Managing people
 
-Members of an organization can see the details of the repositories in that organization and add new repositories to be analyzed by Codacy. Besides this, Codacy only analyzes commits to **private repositories** from contributors who are members of the corresponding organization on Codacy.
+Members of an organization can access the Codacy app to see the dashboards and details of the organization repositories. Depending on their permissions on the Git provider, members can also manage the organization and repositories settings on Codacy.
```

Comment:
Suggestions to:

* Make it more explicit what organization we're referring to (I think it's always a bit confusing whether we're referring to the organization on the Git provider or on Codacy :sweat_smile:)
* Fix a small typo

```suggestion
Members of a Codacy organization can access the Codacy app to see the dashboards and details of the organization repositories. Depending on their permissions on the Git provider, members can also manage the organization and repository settings on Codacy.
```